starport_template/lib/stores/wallets_store.dart

```diff
@@ -58,13 +60,18 @@ class WalletsStore {
 
   set isWalletImporting(bool val) => Action(() => _isWalletImporting.value = val)();
 
+  WalletPublicInfo get selectedWallet => _selectedWallet.value;
+
+  set selectedWallet(WalletPublicInfo val) => Action(() => _selectedWallet.value = val)();
+
   Future<void> loadWallets() async {
     areWalletsLoading = true;
     (await _transactionSigningGateway.getWalletsList()).fold(
       (fail) => loadWalletsFailure.value = fail,
       (newWallets) {
         wallets.clear();
         wallets.addAll(newWallets);
+        selectedWallet = wallets.first;
```

Comment:
this assumes we have a wallet, but in fresh installs it will crash

Reply:
Updated